src/kinetics/Arrhenius.cpp

```diff
@@ -86,11 +71,12 @@ void ArrheniusBase::setRateParameters(
     if (m_A > 0.0) {
         m_logA = std::log(m_A);
     }
+    m_valid = true;
 }
 
 void ArrheniusBase::getRateParameters(AnyMap& node) const
 {
-    if (std::isnan(m_A)) {
+    if (!valid()) {
```

Comment:
I think `isValid()` would be a more descriptive name for this method.

Reply:
As `Reaction::valid` already exists (and the method returns the value of a member variable), the nomenclature of `ReactionRate::valid` is consistent.